packages/react/src/themes/teams/colors.ts

```diff
@@ -1,16 +1,26 @@
-import {
-  ColorPalette,
-  EmphasisColors,
-  NaturalColors,
-  PrimitiveColors,
-  ColorScheme,
-  ColorSchemeMapping,
-  ColorVariants,
-  ContextualColors,
-} from '../types'
-import { Extendable } from '../../types'
-
-export const emphasisColors: ContextualColors = {
+import { ColorPalette, PrimitiveColors, ColorSchemeMapping, ColorVariants } from '../types'
+import { ColorScheme } from 'src/themes/types'
+
+export type TeamsContextualColors = {
+  brand: ColorVariants
```

Comment:
on `ColorVariants` - currently this type is defined as type that any Stardust theme could use. However, we don't have any piece of Stardust code that relies on this structure (actually, good thing, as different themes might use different schemes to organise colors), and Teams is the only theme which uses it. 

I think that it is logical to move this type from common to the types of Teams theme

Reply:
The base theme is using these typings as well. Yes is not required by any react specific file, but still is used in base theme..